paper/paper.md

```diff
@@ -0,0 +1,281 @@
+However, it introduces a new BIDS entity ``atlas-<atlas_label>`` that is used in combination
+  with the ``res-<atlas_scale>`` entity to distinguish imaging and network data derived
+  from different parcellation atlases and scales.
+While the BIDS-Derivatives extension to organize network data is being developed, in which we
+  are actively participating, both structural and functional connectomes generated with CMP3 are
+  saved by default as graph edge lists in ``.tsv`` files, that can be directly analyzed using
+  the \href{https://networkx.org/documentation/stable/tutorial.html}{NetworkX}, a Python library which
+  offers many algorithms and tools to explore graphs and compute local and global network properties.
+Connectivity matrices exported to Matlab as MAT-files and fed to the
+  \href{www.brain-connectivity-toolbox.net}{Brain Connectivity Toolbox}, which is a powerful
+  toolbox containing a large selection of network measures for the characterization of brain
+  connectivity datasets.
+Finally, connectomes can be saved in GraphML format to interface with a lot of general purpose
+  software packages for graph analysis such as \href{www.cytoscape.org}{Cytoscape} or \href{www.gephi.org}{Gephi}.
+Not only this ensures that the connectome files can be opened by
+  the most popular software packages used in this field to perform complex network
+  analyzes, but also this eases the reuse of all outputs in the BIDS ecosystem.
+
+\textbf{Developed with openness, transparency, and good practices in mind.}
+CMP3 is published under the terms of the open source 3-Clause Berkeley Software
+  Distribution (3-Clause BSD) license, which allows unlimited modification, redistribution
+  and commercial use in source and binary forms as long as its copyright notices, and the
+  license's disclaimers of warranty are maintained.
+The source code for CMP3 is hosted at
+  [https://github.com/sebastientourbier/connectomemapper3](https://github.com/sebastientourbier/connectomemapper3),
+  where all bugs and contributions are transparently discussed and managed through issues, and each release is
+  archived to Zenodo [@ZenodoCMP:2021].
+In case of problems, CMP3 has a dedicated forum at
+  [groups.google.com/group/cmtk-users](groups.google.com/group/cmtk-users) where a
+  community of users is active to support and have scientific discussions.
+To be robust to adverse code changes, versions are released through continuous integration building
+  and testing.
+Specifically, this involves testing the installation of the python package, the build of
+  the Docker and Singularity container images, and the execution of the BIDS App via the different container
+  images adopting multiple pipeline configurations, using a sample multi-modal MRI dataset [@Tourbier2020SampleDataset]
+  that has been created for this purpose.
+Doing so, we can guarantee the full functionality of each new released version of CMP3.
```

Comment:
Again I'm not sure but
"functionality of each new**ly** released" sounds more correct to me